inst/sql/modified_update/migrate_coverage.sql

```diff
@@ -5,9 +5,10 @@ coverage_old.id AS coverage_set_id_old,
 coverage_new.id AS coverage_set_id_new
 FROM (SELECT * FROM coverage_set
       WHERE touchstone != '{{{touchstone_new}}}') AS coverage_old
-LEFT JOIN (SELECT coverage_set.* FROM coverage_set
+RIGHT JOIN (SELECT coverage_set.* FROM coverage_set
 JOIN touchstone ON touchstone.id = coverage_set.touchstone
            WHERE touchstone = '{{{touchstone_new}}}'
-           AND coverage_set.gavi_support_level = 'with') AS coverage_new
+           AND (coverage_set.gavi_support_level = 'with'
+             OR coverage_set.gavi_support_level = 'bestminus') )AS coverage_new
```

Comment:
Is this always something we'd want?

Reply:
(If not can you do the same thing as above and start the process of making the support level an argument to `modified_update_calculate` and passing that along to `mu_prepare` and I'll take care of patching the query)

Reply:
Yes, we always want the best estimate as the new coverage, which is mostly 'with' but 'bestminus' for 201210gavi-201303gavi-1.

Reply:
Great, thanks for confirming